s3prl/downstream/asr/hf_hub_templates/README.md

```diff
@@ -0,0 +1,13 @@
+---
+library_name: superb
+benchmark: superb
```

Comment:
i realise it might be a bit redundant to have `superb` appear multiple times, but i need to specify the benchmark and task in order to pick out the correct model repos during inference + evaluation